src/dmd/init.d

```diff
@@ -336,3 +290,51 @@ version (all)
         return false;
     }
 }
+
+
+/****************************************
+ * Copy the AST for Initializer.
+ * Params:
+ *      inx = Initializer AST to copy
+ * Returns:
+ *      the copy
+ */
+Initializer syntaxCopy(Initializer inx)
```

Comment:
Can any of the parameters be made `const`?

Reply:
no